Automated testing fail during debuild

### Type of request

Bug

### OS version

Debian 11 Bullseye

### wmbusmeters version

github master

### Your message goes here

Running 'debuild' to built the .deb, but the self testing fail like this:

```
make[1]: Leaving directory '/home/pere/src/wmbusmeters-new'
   dh_auto_test
        make -j2 test VERBOSE=1
make[1]: Entering directory '/home/pere/src/wmbusmeters-new'
Building 1.11.0-131-gc4e943d with local changes
OK: test internals
OK: Test C1 meters
OK json: Test T1 meters
OK fields: Test T1 meters
OK json: Test S1 meters
OK fields: Test S1 meters
OK: Test mbus
OK: Test ANYID with explicit driver
OK: Test ANYID with auto driver
OK: Test ANYID with auto but no driver found!
OK: Test list-envs and list-fields
OK: Test list-envs and list-fields
OK json: Test non-compliant apator162 meters
OK fields: Test non-compliant apator162 meters
OK json: Test Izars meters
OK: Test listen to all izars
OK: Test aes encrypted telegrams
OK: Test that telegram really is encrypted
OK: Test that failed decryption warning only prints once.
OK: Test shell invocation
OK: Test shell in config file
OK: Test shell environment
OK: Test that normal meterfiles are written
OK: Test that meterfiles with name-id are written
OK: Test that meterfiles with id are written
OK: Test that meterfiles with timestamps are written
OK: Test that meterfiles with timestamps are written
OK: Test normal config files with C1 meters
OK: Test logfile
OK: Test single meter conf file matches any id
OK: Test listen and print any meter heard in logfile
OK: Test listen and print any meter heard on stdout
OK: Test detection of drivers for all telegrams
OK: Test listen to wildcard * id
OK: Test listen to wildcard suffix 8856* id
OK: Test listen to two comma separted ids
OK: Test listen to three comma separted ids
OK: Test listen with negated ids
OK: Test conversions of units using calculations
OK: Test calculations
OK: Test cmdline formulas --calculate_... 
OK: Test meter config formulas calculate_... 
OK: Test date calculations
OK: Test selected fields
OK: Test oneshot
OK: Test wrong keys
OK: Test config4 with added conversions
OK: Test cmdline --overridedevice and --useconfig
OK: Test cmdline --overridedevice without --useconfig
OK: Test cmdline --overridedevice with --useconfig and surplus argument
OK: Test cmdline --overridedevice with --useconfig and surplus arguments
OK: Test cmdline --overridedevice with --useconfig and surplus argument prefixed
OK: Test cmdline --overridedevice with --useconfig and surplus argument infixed
OK: Test additional json from cmdline
OK: Test additional shell envs from cmdline
OK: Test additional json from wmbusmeters.conf and from meter file
OK: Test rtlwmbus starting background script to produce telegrams
OK: Reading binary telegram from stdin
OK: Reading binary telegram from file
OK: Reading rtlwmbus formatted telegrams from stdin
OK: Reading rtlwmbus formatted telegrams from file
OK: Reading rtl433 formatted telegrams from stdin
OK: Reading rtl433 formatted telegrams from file
OK: Test ok serial rawtty telegram on stdin
OK: Test bad serial rawtty telegram with interspersed noise, on stdin
OK: Test duplicates are ignored
OK: Test duplicates are left alone
OK: Test match on dll id
OK: Test match on tpl id
OK fields: Test unix timestamp in fields
OK: Test log timestamps
OK: Test meter driver extra settings
OK: Test hex on commandline
OK: Test hex on commandline with meter
OK: Test invalid hex on commandline
OK: Test hex on stdin
OK: Test auto removal of dll crcs in simulated telegrams
OK: Test broken telegrams
OK: Test rtlwmbus with crc errors
OK: Test timestamps from rtlwmbus
Testing drivers
OK json: driver_abbb23.cc
OK fields: driver_abbb23.cc
OK json: driver_amiplus.cc
OK fields: driver_amiplus.cc
OK json: driver_apator08.cc
OK fields: driver_apator08.cc
OK json: driver_apator162.cc
OK fields: driver_apator162.cc
OK json: driver_apator172.cc
OK fields: driver_apator172.cc
OK json: driver_apatoreitn.cc
OK fields: driver_apatoreitn.cc
OK json: driver_aventieshca.cc
OK fields: driver_aventieshca.cc
OK json: driver_aventieswm.cc
OK fields: driver_aventieswm.cc
OK json: driver_bfw240radio.cc
OK fields: driver_bfw240radio.cc
OK json: driver_c5isf.cc
OK fields: driver_c5isf.cc
OK json: driver_cma12w.cc
OK fields: driver_cma12w.cc
OK json: driver_compact5.cc
OK fields: driver_compact5.cc
OK json: driver_dme_07.cc
OK fields: driver_dme_07.cc
OK json: driver_ebzwmbe.cc
OK fields: driver_ebzwmbe.cc
OK json: driver_ehzp.cc
OK fields: driver_ehzp.cc
OK json: driver_ei6500.cc
OK fields: driver_ei6500.cc
OK json: driver_elf.cc
OK fields: driver_elf.cc
OK json: driver_em24.cc
OK fields: driver_em24.cc
OK json: driver_emerlin868.cc
OK fields: driver_emerlin868.cc
OK json: driver_enercal.cc
OK fields: driver_enercal.cc
OK json: driver_engelmann-faw.cc
OK fields: driver_engelmann-faw.cc
OK json: driver_esyswm.cc
OK fields: driver_esyswm.cc
OK json: driver_eurisii.cc
OK fields: driver_eurisii.cc
OK json: driver_ev200.cc
OK fields: driver_ev200.cc
OK json: driver_evo868.cc
OK fields: driver_evo868.cc
OK json: driver_fhkvdataiii.cc
OK fields: driver_fhkvdataiii.cc
OK json: driver_fhkvdataiv.cc
OK fields: driver_fhkvdataiv.cc
OK json: driver_flowiq2200.cc
OK fields: driver_flowiq2200.cc
OK json: driver_gransystems.cc
OK fields: driver_gransystems.cc
OK json: driver_gwfwater.cc
OK fields: driver_gwfwater.cc
OK json: driver_hydrocalm3.cc
OK fields: driver_hydrocalm3.cc
OK json: driver_hydroclima.cc
OK fields: driver_hydroclima.cc
OK json: driver_hydrodigit.cc
OK fields: driver_hydrodigit.cc
OK json: driver_hydrus.cc
OK fields: driver_hydrus.cc
OK json: driver_iperl.cc
OK fields: driver_iperl.cc
OK json: driver_itron.cc
OK fields: driver_itron.cc
OK json: driver_izar.cc
OK fields: driver_izar.cc
OK json: driver_kamheat.cc
OK fields: driver_kamheat.cc
OK json: driver_kampress.cc
OK fields: driver_kampress.cc
OK json: driver_lansendw.cc
OK fields: driver_lansendw.cc
OK json: driver_lansenpu.cc
OK fields: driver_lansenpu.cc
OK json: driver_lansensm.cc
OK fields: driver_lansensm.cc
OK json: driver_lansenth.cc
OK fields: driver_lansenth.cc
OK json: driver_lse_07_17.cc
OK fields: driver_lse_07_17.cc
OK json: driver_lse_08.cc
OK fields: driver_lse_08.cc
OK json: driver_microclima.cc
OK fields: driver_microclima.cc
OK json: driver_minomess.cc
OK fields: driver_minomess.cc
OK json: driver_mkradio3.cc
OK fields: driver_mkradio3.cc
OK json: driver_mkradio4.cc
OK fields: driver_mkradio4.cc
OK json: driver_mkradio4a.cc
OK fields: driver_mkradio4a.cc
OK json: driver_multical21.cc
OK fields: driver_multical21.cc
OK json: driver_munia.cc
OK fields: driver_munia.cc
OK json: driver_nemo.cc
OK fields: driver_nemo.cc
OK json: driver_omnipower.cc
OK fields: driver_omnipower.cc
OK json: driver_piigth.cc
OK fields: driver_piigth.cc
OK json: driver_pollucomf.cc
OK fields: driver_pollucomf.cc
OK json: driver_q400.cc
OK fields: driver_q400.cc
OK json: driver_qcaloric.cc
OK fields: driver_qcaloric.cc
OK json: driver_qheat.cc
OK fields: driver_qheat.cc
OK json: driver_qheat_55_us.cc
OK fields: driver_qheat_55_us.cc
OK json: driver_qsmoke.cc
OK fields: driver_qsmoke.cc
OK json: driver_qualcosonic.cc
OK fields: driver_qualcosonic.cc
OK json: driver_qwater.cc
OK fields: driver_qwater.cc
OK json: driver_rfmamb.cc
OK fields: driver_rfmamb.cc
OK json: driver_rfmtx1.cc
OK fields: driver_rfmtx1.cc
OK json: driver_sensostar.cc
OK fields: driver_sensostar.cc
OK json: driver_sharky.cc
OK fields: driver_sharky.cc
OK json: driver_sharky774.cc
OK fields: driver_sharky774.cc
OK json: driver_sontex868.cc
OK fields: driver_sontex868.cc
OK json: driver_supercom587.cc
OK fields: driver_supercom587.cc
OK json: driver_topaseskr.cc
OK fields: driver_topaseskr.cc
OK json: driver_tsd2.cc
OK fields: driver_tsd2.cc
OK json: driver_ultraheat.cc
OK fields: driver_ultraheat.cc
OK json: driver_ultrimis.cc
OK fields: driver_ultrimis.cc
OK json: driver_unismart.cc
OK fields: driver_unismart.cc
OK json: driver_vario411.cc
OK fields: driver_vario411.cc
OK json: driver_vario451.cc
OK fields: driver_vario451.cc
OK json: driver_vario451mid.cc
OK fields: driver_vario451mid.cc
OK json: driver_waterstarm.cc
OK fields: driver_waterstarm.cc
OK json: driver_watertech.cc
OK fields: driver_watertech.cc
OK json: driver_weh_07.cc
OK fields: driver_weh_07.cc
OK: Test analyze encrypted (no-key) ctr full telegram
OK: Test analyze encrypted (no-key) ctr compact telegram
OK: Test analyze encrypted (with-key) ctr full telegram
OK: Test analyze encrypted (no-key) ctr compact telegram
localhost [127.0.0.1] 33432 (?) : Connection refused
ERROR: GOT  but expected 123.529
ERROR: Test using netcat to feed rtlwmbus stream using tcp.
localhost [127.0.0.1] 32605 (?) : Connection refused
ERROR: GOT  but expected 123.529
ERROR: Test using netcat to feed hex stream using tcp.
localhost [127.0.0.1] 31574 (?) : Connection refused
ERROR: GOT  but expected 123.529
ERROR: Test using netcat to feed rawtty stream using tcp.
Slower tests...
OK: reading all stdin before closing down
OK: Test config with oneshot
OK: Test config with exitafter
OK: Test config override with oneshot
OK: Test config override with exitafter
RUNNING Test alarms ...
STDERR---------------------------------
[ALARM DeviceInactivity] 4 seconds of inactivity resetting simulations/simulation_alarm.txt simulation (timeout 4s expected mon-sun(00-23) now 2023-03-05 18:21)
(wmbus) successfully reset wmbus device
STDOUT---------------------------------
TMP/OUTPUT-----------------------------
METER =={"media":"cold water","meter":"multical21","name":"Water","id":"76348799","status":"DRY","total_m3":6.408,"target_m3":6.408,"flow_temperature_c":127,"external_temperature_c":19,"current_status":"DRY","time_dry":"22-31 days","time_reversed":"","time_leaking":"","time_bursting":"","timestamp":"2023-03-05T17:21:54Z"}==
METER =={"media":"cold water","meter":"multical21","name":"Water","id":"76348799","status":"DRY","total_m3":6.408,"target_m3":6.408,"flow_temperature_c":127,"external_temperature_c":19,"current_status":"DRY","time_dry":"22-31 days","time_reversed":"","time_leaking":"","time_bursting":"","timestamp":"2023-03-05T17:22:00Z"}==
TMP/ALARM------------------------------
ALARM_SHELL DeviceInactivity [ALARM DeviceInactivity] 4 seconds of inactivity resetting simulations/simulation_alarm.txt simulation (timeout 4s expected mon-sun(00-23) now 2023-03-05 18:21)
---------------------------------------
OK: Test alarms
Some tests failed!
make[1]: *** [Makefile:318: test] Error 1
make[1]: Leaving directory '/home/pere/src/wmbusmeters-new'
dh_auto_test: error: make -j2 test VERBOSE=1 returned exit code 2
make: *** [debian/rules:17: binary] Error 25
dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
debuild: fatal error at line 1182:
dpkg-buildpackage -us -uc -ui failed
```

--- a/deb/control
+++ b/deb/control
@@ -5,8 +5,8 @@ Maintainer: Fredrik Öhrström <oehrstroem@gmail.com>
 Build-Depends: debhelper-compat (= 13)
  , librtlsdr-dev
  , libncurses-dev
- , adduser
  , jq
+ , netcat-openbsd
 Standards-Version: 4.5.1
 Homepage: https://github.com/weetmuts/wmbusmeters
 Vcs-Browser: https://github.com/weetmuts/wmbusmeters
@@ -16,7 +16,8 @@ Rules-Requires-Root: no
 Package: wmbusmeters
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}
-Suggest: rtl-wmbus
+ , adduser
+Suggests: rtl-wmbus
 Description: read wireless and wired M-BUS telegrams from utility meters
  Receives and decodes C1,T1 or S1 telegrams (using
  the wireless or wired mbus protocol) to acquire utility meter

